.github/infrastructure/conformance/azure/conf-test-azure-iothub.bicep

```diff
@@ -18,6 +18,7 @@ param confTestTags object = {}
 var iotHubBindingsConsumerGroupName = '${iotHubName}/events/bindings-cg'
 var iotHubPubsubConsumerGroupName = '${iotHubName}/events/pubsub-cg'
 var iotHubPubsubEhCertificationConsumerGroupName = '${iotHubName}/events/pubsub-eh-certification-cg'
+var iotHubBindingsEhCertificationConsumerGroupName = '${iotHubName}/events/bindings-consumergroup'
```

Comment:
When did you add this? Anything added here won't be applied to the infrastructure automatically and must be applied to me. I was not aware that anything was missing from the infra today?

Reply:
Thanks for catching this. It's not needed anymore, rolling back